sdk/eventhub/azure-eventhub/azure/eventhub/aio/_producer_client_async.py

```diff
@@ -402,7 +405,7 @@ def from_connection_string(
         auth_timeout: float = 60,
         user_agent: Optional[str] = None,
         retry_total: int = 3,
-        transport_type: Optional["TransportType"] = None,
+        transport_type: TransportType = TransportType.Amqp,
```

Comment:
Default is TransportType.Amqp according to docstring. Passing in None results in error after updating uamqp to compare `TransportType.Amqp.value == transport_type.value`, rather than `TransportType.Amqp == transport_type`.

Reply:
Should we handle that AttributeError just in case?

Reply:
I updated the logic so that it does the following:
```
 if transport_type and TransportType.AmqpOverWebsocket.value == transport_type.value or http_proxy:
   # AmqpOverWebsocket logic
else:
   # Amqp logic
```. 
I believe this will account for the transport_type being None (aka. Amqp) and go into the else before the AttributeError is raised.